digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/util/DataFixture.scala

```diff
@@ -92,6 +92,14 @@ object DataFixture {
     println()
   }
 
+  def importRoadAddressesHistory(isDevDatabase: Boolean): Unit = {
```

Comment:
i dont think isdevdatabase boolean is needed

Reply:
yes, you are right!